### ISSUE TITLE
pcre2el package defined both in Spacemacs-editing and Spacenacs-ivy

### ISSUE THREAD

**Issue by zilongshanren** [syl20bnr/spacemacs#5953]
#### Description
(Spacemacs) Warning: More than one init function found for package pcre2el. Previous owner was spacemacs-editing, replacing it with layer spacemacs-ivy.

#### Reproduction guide

Update Spacemacs to the latest develop branch, add `spacemacs-helm` and `spacemacs-ivy` layer.

#### Observed behaviour:
(Spacemacs) Warning: More than one init function found for package pcre2el. Previous owner was spacemacs-editing, replacing it with layer spacemacs-ivy.

#### Expected behaviour:
Start without warning.

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.19
- Spacemacs branch: develop (rev. b772734)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(go windows-scripts spacemacs-helm spacemacs-ivy better-defaults github
    (version-control :variables version-control-diff-tool 'git-gutter  version-control-global-margin t)
    osx latex semantic markdown
    (vinegar :variables vinegar-reuse-dired-buffer t)
    org prodigy search-engine
    (syntax-checking :variables syntax-checking-enable-by-default nil)
    (spell-checking :variables spell-checking-enable-by-default nil)
    yaml python lua html command-log javascript emacs-lisp
    (clojure :variables clojure-enable-fancify-symbols t)
    ranger gtags
    (spacemacs-layouts :variables layouts-enable-autosave t layouts-autosave-delay 300)
    (colors :variables colors-enable-nyan-cat-progress-bar t)
    (git :variables git-magit-status-fullscreen t magit-push-always-verify nil magit-save-repository-buffers 'dontask magit-revert-buffers 'silent magit-refs-show-commit-count 'all magit-revision-show-gravatars nil)
    (ibuffer :variables ibuffer-group-buffers-by 'projects)
    (c-c   :variables c-c  -default-mode-for-headers 'c  -mode)
    (auto-completion :variables auto-completion-enable-sort-by-usage t)
    (shell :variables shell-default-position 'full shell-default-shell 'ansi-term shell-default-term-shell "/bin/zsh")
    (chinese :variables chinese-default-input-method 'wubi chinese-enable-fcitx t chinese-enable-youdao-dict t)
    zilongshanren guanghui)
```

**Comment by StreakyCobra** [comment]
@ralesi `pcre2el` is already provided by `spacemacs-editing` and then should be loaded with `spacemacs-base`. I should have checked this while merging the PR :-/ Does #5918 change ivy's behavior and solve #5812?

**Comment by ralesi** [comment]
@StreakyCobra,  spacemacs-base doesn't load spacemacs-editing (at least not last time I checked),  but I can see that this might be an issue since spacemacs layer loads the spacemacs-base layer

**Comment by ralesi** [comment]
One solution is to port the pcre init to base and just have a postinit with the bindings in spacemacs-editing

**Comment by StreakyCobra** [comment]
@ralesi Indeed, I probably misread the file path, my bad.

The solution you proposed seems to be the most straightforward one, let's ask @syl20bnr if this is what he has in mind for such cases, just to be sure :smiley: ?

**Comment by ralesi** [comment]
@StreakyCobra, I put the fix in if we decide to go that way.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
